src/components/MenuItem.js

```diff
@@ -36,6 +37,8 @@ const defaultProps = {
     wrapperStyle: [],
     style: styles.popoverMenuItem,
     titleStyle: {},
+    shouldShowTitleRightIcon: false,
```

Comment:
`shouldShowTitleRightIcon` this feels just a little bit too wordy... Can we change it to `showRightIcon`?

Reply:
Updated to `shouldShowTitleIcon` and `titleIcon`.

Reply:
Nice